src/scanner.c

```diff
@@ -19,17 +19,24 @@ void tree_sitter_scss_external_scanner_destroy(void *payload) {}
 
 void tree_sitter_scss_external_scanner_reset(void *payload) {}
 
-unsigned tree_sitter_scss_external_scanner_serialize(void *payload, char *buffer) { return 0; }
+unsigned tree_sitter_scss_external_scanner_serialize(void *payload,
```

Comment:
Most of this is `clang_format` and whatever the original code was formatted with disagreeing. The only syntactical change is marked below.